zerver/openapi/zulip.yaml

```
@@ -17600,6 +17600,14 @@ paths:
                           This Zulip server's configured minimum required length for passwords.
                           Necessary for password change UI to show whether the password
                           will be accepted.
+                      password_max_length:
+                        type: integer
+                        description: |
+                          Present if `realm` is present in `fetch_event_types`.
+
+                          This Zulip server's configured maximum required length for passwords.
```

nit:
```suggestion
                          This Zulip server's configured maximum length for passwords.
```

"maximum required" is confusing, as it sounds like it's saying the server will sometimes require a length of this much, or require the length to be at least this much.

---

@gnprice , changed this as per your suggestion.